server/src/config/config.js

```diff
@@ -0,0 +1,8 @@
+/* eslint import/no-dynamic-require: 0 */
+const config = require(`./config.${process.env.NODE_ENV}`);
+
+export const spreadsheetID = () => (config.default.spreadsheetID);
```

Comment:
Please group the related config items together, e.g. under `bugsHistory` so that we know in future why is it there.